docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: TraceQL
+menuTitle: TraceQL
+description: Learn about TraceQL, the query language for traces
```

Comment:
alternative (no strong feelings):

Learn about TraceQL, Tempo's query language for traces

Reply:
Done.